<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>771</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Student Info</string>
     </attribute>
     <widget class="QPushButton" name="addButton1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>290</y>
        <width>211</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteButton1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>340</y>
        <width>211</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="editButton1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>390</y>
        <width>211</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>0</y>
        <width>471</width>
        <height>24</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLineEdit" name="lineSearch"/>
      <widget class="QPushButton" name="searchButton1">
       <property name="text">
        <string>Search ID#</string>
       </property>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>251</width>
        <height>76</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Last Name:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lastName"/>
      </widget>
      <widget class="QSplitter" name="splitter_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>First Name:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="firstName"/>
      </widget>
      <widget class="QSplitter" name="splitter_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Middle Name:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="middleName"/>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>251</width>
        <height>49</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>ID Number:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="ID">
        <property name="placeholderText">
         <string>e.g. 2022-0001</string>
        </property>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Course:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="course">
        <property name="placeholderText">
         <string>e.g. BSCS</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>251</width>
        <height>49</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Gender:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="gender">
        <item>
         <property name="text">
          <string>Male</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Female</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Year Level:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="yrLvl">
        <item>
         <property name="text">
          <string>1st Year</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2nd Year</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3rd Year</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4th Year</string>
         </property>
        </item>
       </widget>
      </widget>
     </widget>
     <widget class="QTableWidget" name="studentT">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>40</y>
        <width>481</width>
        <height>461</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="saveButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>440</y>
        <width>91</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cancelButton1">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>440</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Available Courses</string>
     </attribute>
     <widget class="QSplitter" name="splitter_18">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>390</y>
        <width>731</width>
        <height>24</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLineEdit" name="lineSearch2"/>
      <widget class="QPushButton" name="searchButton2">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_17">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>430</y>
        <width>721</width>
        <height>42</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter_14">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QSplitter" name="splitter_12">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Course Code:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="cC"/>
       </widget>
       <widget class="QSplitter" name="splitter_13">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Course Name:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="cN"/>
       </widget>
      </widget>
      <widget class="QPushButton" name="addButton2">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="saveButton2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>480</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="courseT">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>741</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteButton2">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>480</y>
        <width>221</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="editButton2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>480</y>
        <width>231</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cancelButton2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>480</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
